predict.py

```diff
@@ -238,6 +249,7 @@ def wrapper(self, *args, **kwargs):
     if not USE_SYSTEM_PROMPT:
         # this removes system_prompt from the Replicate API for non-chat models.
         args_to_remove["system_prompt"] = None
+        args_to_remove["prompt_template"] = "{prompt}"
```

Comment:
pretty hidden down here. maybe `"{prompt}"` should be default prompt_template. that's how I did it in `cog-huggingface-template` repo too. fine to have that template always there, even for base models.